src/hardware/vga.cpp

```diff
@@ -483,3 +483,7 @@ void SVGA_Setup_Driver(void) {
 	}
 }
 
+VideoMode VGA_GetCurrentVideoMode() {
+	return vga.draw.render.video_mode;
```

Comment:
Not a big fan of getter functions that don't actually do anything.

Reply:
The value here the API firewall between the VGA module's struct and external modules that want the `VideoMode` (maybe like the rendering code).

We definitely don't want non-VGA module code having to understand the massive `vga.*.*.*` struct layout and poking into it. 

Similar to sdlmain's `GFX_<action>` module calls. We similar don't want the MOUSE_ module to go fetching values deep out of the `sdl.desktop.*.*` structs.

It's a de-spaghettifier :-)

Reply:
That would be fair enough reasoning to add a getter function.  I don't see any usage code in this PR though so there's no way to tell if that reasoning applies here or not.  It's just adding a getter function for no apparent reason.

Reply:
Exactly what @kcgen said. The usefulness of functions are not measured by their line count or their verb prefix 😄 Clear module boundaries is what matters.

I'm not using it here, and I might have ended up not using it later, but inclined to leave it anyway as sooner or later it will be needed. Previously other modules started mucking around in the VGA structs..

Reply:
(But, in general I quite hate getters on objects @weirddan455 so I think I understand where you're coming from. Like these IDEs that auto-magically generate getters/setter for _every single member of a class. That's very questionable. I'm of the objects communicate via _messages_ which are high level and loose, not directly poking into each othernals via getters/setters... That's crap.)

Reply:
Dropped due to popular demand (unused even in my next upcoming PR). Will reinstate when it's needed.